app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java

```diff
@@ -50,6 +54,7 @@ public void onCreate() {
 
         DebotStrategyBuilder builder = new DebotStrategyBuilder.Builder(this)
                 .registerMenu("Clear cache", clearCache)
+                .registerMenu("Notification test ON/OFF", notificationStrategy)
```

Comment:
Sorry, I mean like this.

- If notification test mode is ON, the title should be "Start notification test mode".
- If notification test mode is OFF, the title should be "Stop notification test mode".

Is it possible to check pref here?